### ISSUE TITLE
library comes with a copy of @angular/common

### ISSUE THREAD

**Issue by KristopherWindsor** [mezoistvan/ng2-flatpickr#40]
First I'll cover what I see, then why it's a problem.

When I `npm install ng2-flatpickr` I see that it comes with `node_modules/@angular/common`. This is strange because now I have:
* `<project path>/node_modules/@angular/common`
* `<project path>/node_modules/ng2-flatpickr/node_modules/@angular/common`

<img width="298" alt="screen shot 2018-05-15 at 9 19 59 pm" src="https://user-images.githubusercontent.com/838088/40096506-04d638ec-5886-11e8-83de-ac90739dad16.png">

I spot-checked about 10 other libraries and saw one other doing this. But it seems non-ideal to distribute @angular/common with your library. I guess it might be a build artifact.

This causes issue #32, which I reproduce with Angular 4 when trying to compile for AOT:
```
`Metadata version mismatch for module .../node_modules/ng2-flatpickr/node_modules/@angular/common/common.d.ts, found version 4, expected 3, resolving symbol Ng2FlatpickrModule in .../node_modules/ng2-flatpickr/ng2-flatpickr.d.ts, resolving symbol Ng2FlatpickrModule in .../node_modules/ng2-flatpickr/ng2-flatpickr.d.ts`
```

`node_modules/ng2-flatpickr/node_modules/@angular/common/common.metadata.json` has version 4 metadata, which only works with Angular 5+.
As a workaround, I can simply delete `node_modules/ng2-flatpickr/node_modules`, and then my app will compile.

**Comment by mezoistvan** [comment]
Thanks, I'll look into this. V1.5.x works with Angular v4, so please use that until this is closed.

**Comment by mezoistvan** [comment]
Published 6.0.2 with no dependencies - flatpickr needs to be installed separately from now on. It compiles fine with Angular v6.

**Comment by mezoistvan** [comment]
The problem is that from Angular v6 the metadata version changed. I'll build this package with Angular v4 and publish it as v4.0.0. This way people can use v4 of this with Angular4 and v6 with Angular6.
